Changelog.md

```diff
@@ -1,10 +1,11 @@
 ### 0.8.18 (unreleased)
 
 Language Features:
+ * Add support for ``prevrandao()`` which was introduced in EVM version to "Paris", supplanting ``difficulty`` in the process. Both can be used interchangeably.
```

Comment:
As mentioned on the issue I think this is a bad idea, because the two outputs are expected to be **vastly different** and hence expect different use cases.

Reply:
Well, this is exactly implemented according to the summary after the discussion, so I'm not sure what to tell you here...
https://github.com/ethereum/solidity/issues/13512#issuecomment-1246792479

Reply:
The main point is that we should not *suggest* to use them interchangeably. Supporting both (with a warning on mismatch) is a compromise for not being breaking for hypothetical libraries that want to use it as a source of randomness that can rely on either version - but in general you should still very much use the correct one and not use them interchangeably.